(docs) - Apply edits to automatically flagged errors (#1437)
FormidableLabs/urql: docs/graphcache/errors.md

**This document lists out all errors and warnings in `@urql/exchange-graphcache`.**

> Any unexpected behaviour, condition or error will be marked by an error or warning
> in development, which will output a helpful little message. Sometimes however, this
> message may not actually tell you everything about what's going on.

Reviewer: ```suggestion
Any unexpected behaviour, condition, or error will be marked by an error or warning
in development. This will output as a helpful little message; sometimes, however, this
message may not actually tell you about everything that's going on.
```

Author: Removing the semicolon since the "however" is already a conjunction and hence not omitted, which is what would allow us to use a semicolon.

## (1) Invalid GraphQL document

> Invalid GraphQL document: All GraphQL documents must contain an OperationDefinition
> node for a query, subscription or mutation.

There are multiple places where you're passing in GraphQL documents, either through
methods on `Cache` (e.g. `cache.updateQuery`) or via `urql` using the `Client` or
hooks like `useQuery`.

> Your queries must always contain a main operation, so either a query, mutation or

Reviewer: ```suggestion
Your queries must always contain a main operation, one of: query, mutation, or
```
